riak_repl/manifest.json

```
@@ -2,7 +2,7 @@
   "integration_id": "riak-repl",
   "display_name": "Riak MDC Replication",
   "maintainer": "britt.treece@gmail.com",
-  "manifest_version": "1.0.0",
+  "manifest_version": "1.0.1",
```

```suggestion
  "manifest_version": "1.0.0",
```
This does not need to be updated

---

The manifest version and the code version aren't the same thing?

---

No, it is referring to the version of style of manifest used, as described here: https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file. The only values we support for this field are 1.0.0 and 2.0.0